Assets/Scripts/Properties/Family.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+
+/// <summary>
+/// Represents a container for the Person class.
+/// </summary>
+public class Family
+{
+	/// <summary>
+	/// Constructor of Family with a family name.
+	/// </summary>
+	/// <param name="name">The name of the family.</param>
+	public Family(string name)
+	{
+		this.name = name;
+	}
+
+	/// <summary>
+	/// The name of the family.
+	/// </summary>
+	public string name;
```

Comment:
Instead of "Name" I should've named this something like "FamilyName" to specify it a little bit more